src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ManageContent/ContentBlockLockServiceTests.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.ManageContent;
 
-public class ReleaseContentBlockServiceTests
```

Comment:
I noticed there are still a few references to `ReleaseContentBlockLockEvent` and `ReleaseContentBlockUnlockEvent` in the frontend code - specifically in `releaseContentHub.ts`

Reply:
I'm tempted to leave them for the time being - the reason being that they're specific to Release content alone.  I reckon at the point where we try to use the same content tables for Methodologies as well, we can tackle it then as we make these reusable.